examples/protocols/ethernet/RMII_100BaseTX/main.stanza

```diff
@@ -0,0 +1,246 @@
+#use-added-syntax(jitx)
+defpackage jsl/examples/protocols/ethernet/RMII_100BaseT:
+  import core
+  import jitx
+  import jitx/commands
+  import jitx/parts/query-api
+
+  import jsl/design/settings
+  import jsl/symbols
+  import jsl/landpatterns
+
+  import jsl/si
+  import jsl/bundles
+  import jsl/pin-assignment
+  import jsl/protocols/ethernet/MDI/MDI-100Base-TX
```

Comment:
Typo? Should this be `jsl/protocols/ethernet/MDI/MDI-1000Base-T` instead of `jsl/protocols/ethernet/MDI/MDI-100Base-TX`?

It's surprising that it did not throw any error/warning about this?